src/python_interpreter/mod.rs

```diff
@@ -275,23 +287,24 @@ fn windows_python_info(executable: &Path) -> Result<Option<WindowsPythonInfo>> {
     };
 
     let version_info = str::from_utf8(&python_info.stdout).unwrap();
-    let expr = Regex::new(r"(\d).(\d).(\d+)").unwrap();
-    if let Some(capture) = expr.captures(version_info) {
-        let major = capture.get(1).unwrap().as_str().parse::<usize>().unwrap();
-        let minor = capture.get(2).unwrap().as_str().parse::<usize>().unwrap();
-        let pointer_width = if version_info.contains("64 bit (AMD64)") {
-            64
-        } else {
-            32
-        };
-        Ok(Some(WindowsPythonInfo {
-            major,
-            minor,
-            pointer_width: Some(pointer_width),
-        }))
-    } else {
-        Ok(None)
-    }
+
+    // Split into 3 segments: platform, major, minor by spaces
+    let segments: Vec<&str> = version_info.split_whitespace().collect();
```

Comment:
Is it possible that the platform will ever have whitespace? I think it might be more reliable to re-order the tuple to have major, minor, platform just in case.

We can limit to split into three parts and guarantee this:

```suggestion
    // Split into 3 segments: major, minor, platform by spaces
    let segments: Vec<&str> = version_info.splitn(3, ' ').collect();
    let [major, minor, platform] = pieces.as_slice() else {
        bail!(
            "Unexpected output for Python version info from {}: '{}'",
            executable.display(),
            version_info
        );
    };
    // can then parse each substring
```